openedx/features/edly/api/urls.py

```diff
@@ -0,0 +1,5 @@
+from django.conf.urls import include, url
+
+urlpatterns = [
```

Comment:
@iam-mhaseeb @farhan Do we need to put these API URL's behind the mobile feature flag?

```python
if settings.FEATURES.get('ENABLE_MOBILE_REST_API'):
    urlpatterns += [
        url(r'^api/mobile/(?P<api_version>v(1|0.5))/', include('mobile_api.urls')),
    ]

```

Reply:
@zubair-arbi In might sight, we should because edX mobile API's are also behind this feature flag. Leaving it without a feature flag would let this feature to work but other mobile API's of edX won't work.

Reply:
It means that if `ENABLE_MOBILE_REST_API` feature flag is not set then all the mobile API URL's will give 404.

Reply:
Done